pyproject.toml

```diff
@@ -30,6 +30,7 @@ bleak-retry-connector = ">=2.3.0"
 orjson = ">=3.7.8"
 async-timeout = "^4.0.2"
 chacha20poly1305 = "^0.0.3"
+aiohttp = ">=3.8.3"
```

Comment:
I thought this was just a dev dependency (to get asyncio support in pytest) - do we use it for something else now?

Reply:
It should just be dev.

Reply:
fixed. good catch